content/tutorials/manage-the-rippled-server/configuration/configure-history-sharding.md

```diff
@@ -11,9 +11,9 @@ To configure your `rippled` to store shards of ledger history, complete the foll
 Before you configure your `rippled` server to store history shards, you must decide how much disk space to allocate to the history shard store. This also affects how much history you keep in the default ledger store. You should consider the following when deciding what size to configure your shard store:
 
 - The ledger store (defined by the `[node_db]` stanza) is separate from the history shard store. The ledger store is required for all servers and _must_ contain a range of recent history, defined by how many ledgers to keep available in the `online_delete` parameter. (The default configuration stores the most recent 2000 ledgers.)
-    - If you keep at least 2^15 ledgers (32768) in the ledger store, you can efficiently import chunks of recent history from the ledger store into the shard store.
-- The history shard store (defined by the `[shard_db]` stanza) is only required for storing history shards. The configuration stanza should be omitted from servers that do not store history shards. The size of the history shard store is defined in gigabytes in the `max_size_gb` parameter; the server attempts to use as much of this space as possible to store complete shards.
-- A shard consists of 2^14 ledgers (16384) and occupies approximately 200 MB to 4 GB based on the age of the shard. Older shards are smaller because there was less activity in the XRP Ledger at the time.
+    - If you keep at least 2<sup>15</sup> ledgers (32768) in the ledger store, you can efficiently import chunks of recent history from the ledger store into the shard store.
+- The history shard store (defined by the `[shard_db]` stanza) is only required for storing history shards. The configuration stanza should be omitted from servers that do not store history shards. The size of the history shard store is defined in gigabytes in the `max_size_gb` parameter; the server attempts to use as much of this space as possible to store complete shards. The history shard store _MUST_ be stored on a solid-state disk or similar fast media. Traditional spinning hard disks are insufficient.
+- A shard consists of 2<sup>14</sup> ledgers (16384) and occupies approximately 200 MB to 4 GB based on the age of the shard. Older shards are smaller because there was less activity in the XRP Ledger at the time.
```

Comment:
The addition capitalizes _MUST_ but other sections say _must_. Is this deliberate? 

> ... all servers and _must_ contain a range of recent history
> The history shard store _MUST_ be stored on a solid-state disk ...
> The history shard store and the ledger store _must_ be stored at different file paths.

etc.

Reply:
Btw, this bullet seems to be missing something .. maybe "the"?

```
< The time to acquire a shard, number of file handles needed by the rippled server, and memory cache usage is directly affected by the size of the shard.
---
> The time to acquire a shard, _the_ number of file handles ... , and _the_ memory cache usage ...

```

Reply:
I think I prefer the sentence as-is. I like, "The a, b, and c" over "The a, the b, and the c". But I appreciate the suggestion.